src/lib.rs

```diff
@@ -116,10 +116,10 @@ mod verify {
 }
 
 #[cfg(feature = "async")]
-mod tls_connection;
+mod asynch;
```

Comment:
Is the name really `asynch`? I would have expected `async`, but maybe that conflicts with the keyword?

Reply:
Yeah, it conflicts. The `asynch` is used in embedded-io as well, so I thought I'd follow that new "convention".